woke.yml

```diff
@@ -25,3 +26,11 @@
   tags: tinycount
   roles:
     - tinycount
+
+- name: Setup netdata
+  hosts: tinycount
```

Comment:
This should change to `wokeserver`, right?

Reply:
Yeah, when it's ready to merge. I'm just shortcutting with `tinycount` for now since I'm only testing on `get.woke.net`.